### ISSUE TITLE
put_object does not work for geo

### ISSUE THREAD

**Issue by elbow-jason** [elbow-jason/dgraph_ex#37]
** (FunctionClauseError) no function clause matching in DgraphEx.Field.put_object/2

    The following arguments were given to DgraphEx.Field.put_object/2:

        # 1
        %DgraphEx.Field{count: false, default: nil, facets: nil, index: [:geo],
     label: nil, model: nil, object: nil, predicate: :geo_center, reverse: false,
     subject: :land, type: :geo, virtual: false}

        # 2
        [-112.44615353350031, 33.35600630797468]

    Attempted function clauses (showing 8 out of 8):

        def put_object(%DgraphEx.Field{type: :int} = field, value)
        def put_object(%DgraphEx.Field{type: :string} = field, value) when is_binary(value)
        def put_object(%DgraphEx.Field{type: :date} = field, value)
        def put_object(%DgraphEx.Field{type: :datetime} = field, value)
        def put_object(%DgraphEx.Field{type: :uid} = field, %DgraphEx.Expr.Uid{} = uid)
        def put_object(%DgraphEx.Field{type: :uid} = field, uid) when is_atom(uid)
        def put_object(%DgraphEx.Field{type: :uid_literal} = field, %DgraphEx.Expr.Uid{} = uid)
        def put_object(%DgraphEx.Field{type: :uid_literal} = field, <<""::binary(), uid::binary()>>)

    (dgraph_ex) lib/dgraph_ex/field.ex:120: DgraphEx.Field.put_object/2
    (elixir) lib/enum.ex:1255: Enum."-map/2-lists^map/1-0-"/2
    (elixir) lib/enum.ex:1255: Enum."-map/2-lists^map/1-0-"/2
    (dgraph_ex) lib/dgraph_ex/vertex.ex:88: DgraphEx.Vertex.populate_fields/3
    (dgraph_ex) lib/dgraph_ex.ex:15: DgraphEx.set/3
    (dgraph_ex) lib/dgraph_ex/repo.ex:56: DgraphEx.Repo.insert/1

**Comment by elbow-jason** [comment]
fix in commit  c26d336

**Comment by elbow-jason** [comment]
```elixir
  test "put_object works with a geo field" do
    some_location = [-112.44615353350031, 33.35600630797468]
    the_field = %Field{
      subject:    %Uid{value: "1234", type: :literal},
      type:       :geo,
      predicate:  :location,
    }
    |> Field.put_object(some_location)
    assert the_field.object == some_location
  end
```
